### PR TITLE
pin version, bump certifi

### PR DESCRIPTION
**Pull request by NPann**
Pin dependencies, Update lock file, Address the certifi vulnerability alert

### CODE REVIEW

File: code/Pipfile [hunk 1]
```diff
@@ -4,13 +4,13 @@ verify_ssl = true
...
 
 [packages]
-flywheel-sdk = "==15.5.0"
-jinja2 = "*"
-flake8 = "*"
-flake8-colors = "*"
-pyyaml = "*"
-pydantic = "*"
-typing-extensions = "*"
+flywheel-sdk = "==15.8.0"
```
Comment (jiavila) [inline code/Pipfile:7]: Can we bump this up to a higher version, one that can use python 3.10 w/o any issues with collections. See https://flywheelio.zendesk.com/agent/tickets/7599

**Review by jiavila — COMMENTED** [review]
everything else is fine, should work OK. For the pipfile, I'm assuming that you regenerated it so that the hash matches these updated versions?

File: code/Pipfile (hunk 1 above)
Comment (NPann) [reply code/Pipfile]: Added

**Comment by jiavila** [comment]
lgtm!

**Review by jiavila — APPROVED** [review]
lgtm!

**Pull request merged**